ref/net46/Microsoft.Build/Microsoft.Build.cs

```diff
@@ -500,10 +500,12 @@ namespace Microsoft.Build.Evaluation
 {
     public partial class GlobResult
     {
-        public GlobResult(Microsoft.Build.Construction.ProjectItemElement itemElement, string glob, System.Collections.Generic.IEnumerable<string> excludes) { }
+        internal GlobResult() { }
         public System.Collections.Generic.IEnumerable<string> Excludes { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } }
-        public string Glob { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } }
```

Comment:
I could unbreak it by:
- keeping the constructor public and overloading it
- mark Glob as deprecated and have it return the first glob. After this change, there is one GlobResult per item element with globs. Before, there was one GlobResult per glob fragment, so one item element could have produced multiple GlobResults.

As an alternative, I could keep the old code and have the new variants as GetAllGlobs2 and GlobResult2 :)

I searched on Github and Index and found no other usage other than CPS. So breaking it looks like a viable option, since everything else introduces baggage.